src/lf_mask.rs

```diff
@@ -157,14 +180,17 @@ fn mask_edges_inter(
     let t_dim = &dav1d_txfm_dimensions[max_tx as usize];
 
     // See [`decomp_tx`]'s docs for the `txa` arg.
-    let mut txa = Align16([[[[0; 32]; 32]; 2]; 2]);
+    let mut txa: MaybeUninit<Align16<[[[[u8; 32]; 32]; 2]; 2]>> = MaybeUninit::uninit();
```

Comment:
Why can't we do `Align16<[[[[MabybeUninit<u8>; 32]; 32]; 2]; 2]>`?  Or at least `Align16<[[[MabybeUninit<[u8; 32]>; 32]; 2]; 2]>` since the 3 outer dimensions we index normally (which can be safe) and we only pass the inner one to `set_raw`.

Reply:
I guess I could if I transmuted things. I didn't do this because the APIs for handling uninitialized arrays aren't stable yet (https://github.com/rust-lang/rust/issues/96097). But... it would be much cleaner. I'll do that instead.